### PR TITLE
Added featured posts.

### PR DESCRIPTION
**Pull request by tan-90**
Added `FEATURED` switch to CMS.
Added featured posts section to blog landing page.
Added landing page for featured posts.

Closes WEB-76

### CODE REVIEW

**Comment by Lisenish** [comment]
@tan-90 Please update sync develop branch

File: src/templates/blog.module.scss [hunk 1]
```diff
@@ -1,106 +1,3 @@
-.grid {
```
Comment (Lisenish) [inline src/templates/blog.module.scss:1]: Nice :)

File: src/templates/blog.tsx [hunk 2]
```diff
@@ -2,110 +2,87 @@ import { graphql, Link } from 'gatsby'
...
-          )}
+          <PaginationNav currentPage={currentPage} numPages={numPages} basePath='/blog/' />
         </Container>
       </Section>
     </Layout>
   )
 }
 
 export const pageQuery = graphql`
+  fragment PostData on MarkdownRemarkConnection {
```
Comment (Lisenish) [inline src/templates/blog.tsx:41]: 👍 

I believe you can actually even separate fragment to another exported const with arbitrary name just to have only page query in pageQuery const

File: src/templates/blog-featured.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    }
+  }
+`
+
+interface BlogFeaturedContext {
+  currentPage: number
+  numPages: number
+}
+
+const dateFormatter = new Intl.DateTimeFormat('en-US', { month: 'long', day: 'numeric', year: 'numeric' })
```
Comment (Lisenish) [inline src/templates/blog-featured.tsx:78]: Just left a comment in another review that we probably should have all formatter in one common place

**Review by Lisenish — CHANGES_REQUESTED** [review]

**Comment by Lisenish** [comment]
@tan-90 I think we should delete our test post and feature some existing one, so we don't ship these test page on production :)

**Review by Lisenish — APPROVED** [review]

**Comment by Lisenish** [comment]
@tan-90 For some reason there are old link in blog cards? Something merged incorrectly or we need another sync with develop?

**Comment by tan-90** [comment]
> @tan-90 For some reason there are old link in blog cards? Something merged incorrectly or we need another sync with develop?

They're like that on develop as well. Maybe it never changed? I think me and Natanael were working in parallel at the time. I'll fix it now.

**Comment by Lisenish** [comment]
@tan-90 It's good that you fixed that ones, but I actually meant blog posts cards not use cases, looks like they use old filename urls instead of metaurl

**Pull request merged**